api/utilities.py

```diff
@@ -13,4 +13,46 @@ def facebook_graph(request):
     page_id= os.getenv('PAGE_ID')
     page_message="Hello!"
     page.put_object(page_id,"feed",message=page_message)
-    return HttpResponse("Successfully posted to facebook!")
\ No newline at end of file
+    return HttpResponse("Successfully posted to facebook!")
+
+def instagram_graph(description, image_url):
+    load_dotenv()
+    access_token = os.getenv('ACCESS_TOKEN_FB')
+    
+    main_url = "https://graph.facebook.com/v13.0/"
+    headers = {
+    "Authorization": "Bearer " + access_token
+    }
+
+    ## Gets the facebook id of the user's page using the access token
+    url = main_url + "me/accounts"
+    response = get(url, headers=headers).json()
```

Comment:
Should we wrap the response into a try catch block to catch if the IG api runs into any errors or if we hit the rate limit of the day somehow? I think we should have a redundancy that prevents our server from crashing if the IG api is down or not working for whatever reason!